### DISCUSSION TITLE
Creating an HTML foldable / collapsible section on a page

### DISCUSSION THREAD

**Discussion by sunrays2993** [GetPublii/Publii#1796] [Help]
I have tried a few ways that I have found online for creating a foldable section for I can hide text unless someone wants to expand it, but I cannot get them to work.

They involve entering some new CSS, which I have put into the "Tools & Plugins > Custom CSS", then placing some HTML onto the page. 

I am using the Editorial 2 theme.

Has anyone has experience whit this?

**Comment by AKickIndieNutz** [comment]
Hi, not quite sure what you are after but on my site I have an archive Page which just contains html and displays the Year and a subheading of Month hidden below that, when clicked on reveals each post for that month. The code is something like:

<p><span style="color: #f7ecad;">Year</span></p>
<div class="post__toc">
	<details>
		<summary><span style="color: #e03e2d;">Month Title</span></summary>
		<summary><span style="color: #ffff00;">Actual Month</span></summary>
		<a href="#INTERNAL_LINK#/post/123" target="_blank" rel="noopener noreferrer">Post Title 1 Here</a><a> <br></a> <a href="#INTERNAL_LINK#/post/456" target="_blank" rel="noopener noreferrer">Post Title 2 Here</a><a> <br></a>

Not sure if it will be of help for what you are after but fingers crossed ;-)

**Reply by bjazmoore** [reply]
Are you trying to create an accordion?

**Comment by AKickIndieNutz** [comment]
```
`<p><span` style="color: #f7ecad;">Year</span></p>
<div class="post__toc">
	<details>
		<summary><span style="color: #e03e2d;">Month Title</span></summary>
		<summary><span style="color: #ffff00;">Actual Month</span></summary>
		<a href="#INTERNAL_LINK#/post/xxx" target="_blank" rel="noopener noreferrer">Post Title 1 Here</a><a> <br></a> <a href="#INTERNAL_LINK#/post/xxx" target="_blank" rel="noopener noreferrer">Post Title 2 Here</a><a> <br></a>`
```

**Reply by candidexmedia** [reply]
You're missing a closing `</details>` tag. Try this:

```html
<details>
  <summary><span style="color: #e03e2d;">Month Title</span></summary>
  <summary><span style="color: #ffff00;">Actual Month</span></summary>
  <a href="#INTERNAL_LINK#/post/xxx" target="_blank" rel="noopener noreferrer">Post Title 1 Here</a><a> <br></a> <a href="#INTERNAL_LINK#/post/xxx" target="_blank" rel="noopener noreferrer">Post Title 2 Here</a><a> <br></a>
</details>
```

Also, you should only have one `<summary>` per `<details>` disclosure element: https://stackoverflow.com/questions/72065393/is-it-possible-to-include-more-than-one-summary-elements-inside-of-a-detail-elem

Having more than one doesn't make much sense, [semantically](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details).
